app/src/main/java/dev/arkbuilders/arkmemo/utils/SVGUtils.kt

```diff
@@ -0,0 +1,172 @@
+    }
+
+    private fun createPaths(): Stack<DrawPath> {
+        val paths = Stack<DrawPath>()
+        if (pathData.isNotEmpty()) {
+            var path = AndroidDrawPath()
+            var commandsString = pathData
+            val numberOfCommands = commandsString.filter {
+                it.isLetter()
+            }.length
+            Log.d("svg-utils", "$numberOfCommands commands available")
+            for (index in 1..numberOfCommands) {
+                when (commandsString.substringAfter(SPACE).substringBefore(SPACE)) {
+                    MOVE_TO -> {
+                        path = AndroidDrawPath()
+                        val point = commandsString.substringAfter("$MOVE_TO$SPACE")
+                            .substringBefore(SPACE)
+                        val pointList = point.split(COMMA)
+                        val x = pointList[0].toFloat();
+                        val y = pointList[1].toFloat()
+                        Log.d("svg-utils", "move to $point")
+                        commandsString = commandsString.removePrefix("$SPACE$MOVE_TO$SPACE$point")
+                        path.moveTo(x, y)
+                        paths.add(DrawPath(path, paint))
+                    }
+
+                    ABS_QUAD_TO -> {
+                        val point1 = commandsString.substringAfter("$ABS_QUAD_TO$SPACE")
+                            .substringBefore(SPACE)
+                        commandsString =
+                            commandsString.removePrefix("$SPACE$ABS_QUAD_TO$SPACE$point1")
+                        val point2 = commandsString.substringAfter(SPACE).substringBefore(SPACE)
+                        commandsString = commandsString.removePrefix("$SPACE$point2")
+                        val point1List = point1.split(COMMA)
+                        val point2List = point2.split(COMMA)
+                        val x1 = point1List[0].toFloat();
+                        val y1 = point1List[1].toFloat()
+                        val x2 = point2List[0].toFloat();
+                        val y2 = point2List[1].toFloat()
+                        Log.d("svg-utils", "quad to $point1 $point2")
+                        path.quadTo(x1, y1, x2, y2)
+                    }
+
+                    ABS_LINE_TO -> {}
+                }
+            }
+        }
+        return paths
+    }
+
+    companion object {
+        fun parse(path: Path) = SVG().apply {
+            val xmlParser = Xml.newPullParser()
+            xmlParser.apply {
+                setInput(path.reader())
+                var event = xmlParser.eventType
+                while (event != XmlPullParser.END_DOCUMENT) {
+                    val tag = xmlParser.name
+                    when (event) {
+                        XmlPullParser.START_TAG -> {
+                            when (tag) {
+                                "svg" -> { viewBox = getAttributeValue("", Attributes.VIEW_BOX) }
+                                "path" -> {
```

Comment:
I could see that `svg` and `path` are being used in several classes.
Let's try avoiding hard code and use constants instead.